cypress/e2e/buy-shirt.cy.ts

```diff
@@ -21,29 +21,18 @@ describe("Buy a t-shirt", () => {
     menuContentPage.visitMenuContentPage();
     menuContentPage.goToTShirtMenu();
 
-    // Add chosen product to cart from product list
     productsListPage.addToCart();
     productsListPage.checkout();
 
-    // Shopping cart summary
     shoppingCartPage.checkoutCart();
 
-    // Authentication - Login
     loginPage.login("aperdomobo@gmail.com", "WorkshopProtractor");
 
-    // Address - Already set
-    addresStepPage.checkout();
+    addressStepPage.checkout();
 
-    // Shipping: Agree to terms and proceed
     shippingStepPage.checkout();
 
-    // Payment by bank wire
     paymentStepPage.payByBank();
-
-    // Confirm order
-    cy.get("#cart_navigation > .button").click();
-
-    // Assert completed order message on page
-    cy.get("#center_column > div.box > p > strong").should("have.text", "Your order on My Store is complete.");
+    paymentStepPage.getConfirmationMessage();
```

Comment:
Here I would expect to have the assertion:
`paymentStepPage.getConfirmationMessage().should("have.text", "Your order on My Store is complete.");`